packages/contracts-bedrock/src/cannon/PreimageOracle.sol

```diff
@@ -19,7 +19,7 @@ contract PreimageOracle is IPreimageOracle {
     /// @notice The duration of the large preimage proposal challenge period.
     uint256 internal immutable CHALLENGE_PERIOD;
     /// @notice The minimum size of a preimage that can be proposed in the large preimage path.
-    uint256 internal immutable MIN_LPP_SIZE_BYTES;
+    uint256 public immutable MIN_LPP_SIZE_BYTES;
```

Comment:
nit: Per style guide, immutables should be internal in favor of explicit getters for them.

Reply:
Can do

Reply:
For the record, we already have `minProposalSize()`

Reply:
Oops too late, will revert :)